test/create_db.sh

```diff
@@ -37,12 +42,11 @@ for svc in $SERVICES; do
 		USERS_SQL=test/${svc}_db_users.sql
 		if [[ -f $USERS_SQL ]]; then
 			if [[ $MYSQL_CONTAINER ]]; then
-				sed -e "s/'localhost'/'127.0.0.1'/g" < $USERS_SQL | \
-					mysql $dbconn -D $db || \
-					die "unable to add users to ${db}"
+				sed -e "s/'localhost'/'172.%'/g" < $USERS_SQL | \
```

Comment:
Any reasons to use 172.% pattern rather than just %? Given that the mysql container runs with MYSQL_ALLOW_EMPTY_PASSWORD it cannot be used as a way to provide any real security benefits like preventing accidental exposure to the internet of a container running with ipv6 enabled and a routable ipv6 address. On the other hand 172.% assumes too much about docker networking details and may need to change on future docker updates.